external/Microsoft.SqlTools.CoreServices/Connection/ConnectionServiceCore.cs

```diff
@@ -220,19 +220,12 @@ public virtual bool TryFindConnection(string ownerUri, out ConnectionInfo connec
         public ConnectionCompleteParams ValidateConnectParams(ConnectParams connectionParams)
         {
             string paramValidationErrorMessage;
-            if (connectionParams == null)
```

Comment:
is this code being used? I've never touched this library before.

Reply:
Not sure if it's being used, but it appears to be identical to ServiceLayer's function, if those changes need to be reverted, I'll be happy to do so.

Reply:
Reverted it as it appears to be external and the string localization tool does not recognize it currently.